src/main/java/org/apache/maven/plugins/enforcer/RequireLicense.java

```diff
@@ -0,0 +1,59 @@
+package org.apache.maven.plugins.enforcer;
+
+import org.apache.maven.enforcer.rule.api.EnforcerRule;
+import org.apache.maven.enforcer.rule.api.EnforcerRuleException;
+import org.apache.maven.enforcer.rule.api.EnforcerRuleHelper;
+import org.apache.maven.model.License;
+import org.apache.maven.plugin.logging.Log;
+import org.apache.maven.project.MavenProject;
+import org.codehaus.plexus.component.configurator.expression.ExpressionEvaluationException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Manuel Recena
+ * @since TODO
+ */
+public class RequireLicense implements EnforcerRule {
+
+    public void execute(EnforcerRuleHelper helper) throws EnforcerRuleException {
+        Log log = helper.getLog();
+        try {
+            MavenProject project = (MavenProject) helper.evaluate( "${project}" );
+            if (!project.getPackaging().equals("pom")) {
+                List<License> licenses = getLicenses(project);
+                if (licenses.size() == 0) {
+                    throw new EnforcerRuleException("You must set a license for this project");
```

Comment:
Just "The license is not set in this project or in it's parents" ?

Reply:
Sure but is _ in its parents?_, right?

Reply:
At the end, where is defined the license is not the critical part but having a license.